src/ctelnet.cpp

```diff
@@ -928,9 +928,9 @@ std::tuple<QString, int, bool> cTelnet::getConnectionInfo() const
     }
 }
 
-void cTelnet::processTelnetCommand(const std::string& command)
```

Comment:
I'd already run into this issue when prototyping code locally but in this case I resorted to switching the class member to be called `mCommand` instead of renaming the one local to this method - that is more consistent with the prefixing of class members that I like to think we try and do.